cfn-guard-rulegen-lambda/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cfn-guard-rulegen-lambda"
+version = "0.1.0"
+authors = ["John Capps <jocapp@amazon.com>"]
```

Comment:
I would remove the 'authors' line.  Your email will get spammed into oblivion.  Your git profile will be able to id your contribution without it anyway.